index.html

```diff
@@ -4975,6 +4975,621 @@ <h2 id="AudioWorkerParamDescriptor">
           </section>
         </section>
       </section>
+      <section>
+        <h2 id="AudioWorklet">
+          The <dfn>AudioWorklet</dfn> interface
+        </h2>
+        <p>
+          The AudioWorklet object allows importing user-supplied Javascript code
+          that runs on the audio rendering thread. This processing mechanism
+          ensures the aligned execution of the script code with other built-in
+          <a>AudioNode</a>s in the audio graph. The AudioWorklet object imports
+          and stores the definition of custom audio processor and then an
+          <a>AudioWorkletNode</a> object can be constructed based on the
+          processor definition. Note that the registered definition in an
+          AudioWorklet is valid for the entire global scope, therefore any
+          <a>AudioContext</a> in the same global scope can be used to create the
+          custom node.
+        </p>
+        <p>
+          An AudioWorklet object provides the capability to import module
+          scripts into its associated <a>AudioWorkletGlobalScope</a>. The user
+          agent can then create <a>AudioWorkletProcessor</a> classes registered
+          on the <a>AudioWorkletGlobalScopes</a> and construct their instances.
```

Comment:
Move the "s" of AudioWorkletGlobalScope outside the tag.  The link won't work as is.